ui/app/AppLayouts/Profile/Sections/MyProfileContainer.qml

```diff
@@ -58,12 +58,12 @@ Item {
 
         StyledText {
             id: pubkeyText
-            text: address
+            text: ensName !== "" ? username : pubkey
```

Comment:
Will the user still be able to see the pubkey somewhere in the case text is set to username?

Reply:
Yes. the chat key is available too on that screen

![image](https://user-images.githubusercontent.com/1106587/89668722-3b5ac380-d8ac-11ea-8af2-1c2f527b6fe1.png)